syncback/krsync.sh

```diff
@@ -28,27 +28,31 @@ install_rsync_tilt() {
     set +e # disable exit on error
     # $K8S_OBJECT is intentionally not quoted here so -n/-c flags can be
     # parsed by kubectl
+    # shellcheck disable=SC2086
     kubectl exec $K8S_OBJECT -- ls "$RSYNC_TILT_PATH" > /dev/null 2>&1
     exit_code="$?"
     set -e
     if [[ "$exit_code" != "0" ]]; then
-        # shellcheck disable=SC2002
         echo "installing tilt's rsync"
-        cat "$KRSYNC_TAR_PATH" | kubectl exec -i $K8S_OBJECT -- tar -xf - -C $(dirname $RSYNC_TILT_PATH)
+        # shellcheck disable=SC2086,SC2002
+        cat "$KRSYNC_TAR_PATH" | kubectl exec -i $K8S_OBJECT -- tar -xf - -C "$(dirname $RSYNC_TILT_PATH)"
     fi
 }
 
 find_rsync_path() {
     local rsync_path=--rsync-path=$RSYNC_TILT_PATH
     # see if we already have a compatible rsync on the container
+    # shellcheck disable=SC2086,SC2181
     rsync_output=$(kubectl exec $K8S_OBJECT -- rsync --version 2> /dev/null)
+    # shellcheck disable=SC2181
     if [ $? -eq 0 ]; then
         detected_rsync_version=$(echo "$rsync_output" | head -1 | awk '{ print $3 }')
-        if [[ "$detected_rsync_version" > 3 ]]; then
```

Comment:
can you revert this? i think we did it deliberately this way because the alphabetical sort will work fine for what we want here

Reply:
done